### PR TITLE
Proposed changes for IETF 117

### LINKED ISSUE #41: Security Considerations
Provide text for the security considerations section

### PR DESCRIPTION
**Pull request by italobusi**
Updated document for IETF 117:
- Restructure appendices A and B: fix https://github.com/IETF-TEAS-WG/actn-poi/issues/29
- Updated text on L3-TE Topology: fix https://github.com/IETF-TEAS-WG/actn-poi/issues/85
- Added standard reference for TI-LFA: fix https://github.com/IETF-TEAS-WG/actn-poi/issues/95
- Added description for the configuration of the the minimum number of active member links required to consider the LAG link as being up : fix https://github.com/IETF-TEAS-WG/actn-poi/issues/98
- Added text describing the support for DWDM networks with mixed fixed and flexible grids: fix https://github.com/IETF-TEAS-WG/actn-poi/issues/100
- Added text describing the gap on reporting muxponder's constraints: fix https://github.com/IETF-TEAS-WG/actn-poi/issues/108
- Updated contact information for Anton, Daniele and Gabriele: fix https://github.com/IETF-TEAS-WG/actn-poi/issues/109
- Added security considerations: fix #41

Co-authored-by: danielkinguk <dk@danielking.net>

### CODE REVIEW

File: draft-ietf-teas-actn-poi-applicability.txt [hunk 1]
```diff
@@ -2694,24 +2719,42 @@ Internet-Draft                 ACTN POI                   November 2022
...
+      but not yet defined: this gap has been identified in section 5.3;
+
+   o  an attribute to configure the minimum number of active member
+      links required to consider the LAG link as being up is missing
+      from the topology model defined in [CLIENT-TOPO]: this gap has
+      been identified in section 5.2.1;
+
+   o  a mechanism to report client connectivity constraints imposed by
+      some muxponder design: this gap has been identified in appendix
+      Error! Reference source not found..
```
Comment (jbouqui153) [inline draft-ietf-teas-actn-poi-applicability.txt:2734]: cross-reference to the appendix A.3 not working

File: draft-ietf-teas-actn-poi-applicability.txt [hunk 2]
```diff
@@ -1332,25 +1324,46 @@ Internet-Draft                 ACTN POI                    January 2023
...
+Internet-Draft                 ACTN POI                       June 2023
 
 
+   o  A mixed fixed and flexible grid DWDM network topology can be
+      reported using either the Flexi-grid Topology model or both WSON
+      and Flexi-grid topology models.
+
+   Clarifying how both WSON and Flexi-grid topology models could be used
+   together (through multi-inheritance) has been identified as a gap.
+
```
Comment (sergiobelotti) [inline draft-ietf-teas-actn-poi-applicability.txt:1360]: Just a suggestion to add a reference to the concept of multi-inheritance to whom is not familiar with these models. (maybe RFC 8345) or draft-busi-teas-te-topology-profile

File: draft-ietf-teas-actn-poi-applicability.txt [hunk 3]
```diff
@@ -2492,6 +2546,15 @@ Internet-Draft                 ACTN POI                    January 2023
...
    single-domain Ethernet link(s) being created, as described above.
 
+   When a new LAG link is created, it is also recommended to configure
+   the minimum number of active member links required to consider the
+   LAG link as being up. For example, a LAG link with three members can
+   be considered up when only one member link fails and down when at
+   least two member links fail.
+
+   This configuration is missing in [CLIENT-TOPO] and this is identified
+   as a gap in section 6.
```
Comment (sergiobelotti) [inline draft-ietf-teas-actn-poi-applicability.txt:2556]: Just to be clear: the model is read only and the intention is to extend that with write part for configuration?

File: draft-ietf-teas-actn-poi-applicability.txt [hunk 4]
```diff
@@ -2640,24 +2719,42 @@ Internet-Draft                 ACTN POI                    January 2023
...
+      but not yet defined: this gap has been identified in section 5.3;
+
+   o  an attribute to configure the minimum number of active member
+      links required to consider the LAG link as being up is missing
+      from the topology model defined in [CLIENT-TOPO]: this gap has
+      been identified in section 5.2.1;
+
+   o  a mechanism to report client connectivity constraints imposed by
+      some muxponder design: this gap has been identified in appendix
+      Error! Reference source not found..
```
Comment (sergiobelotti) [inline draft-ietf-teas-actn-poi-applicability.txt:2734]: same comment as Jeff

**Review by sergiobelotti — APPROVED** [review]

File: draft-ietf-teas-actn-poi-applicability.txt (hunk 2 above)
Comment (italobusi) [reply draft-ietf-teas-actn-poi-applicability.txt:1360]: **2023-07-04 TEAS Call**

Agreed to add an informative reference to draft-busi-teas-te-topology-profile

File: draft-ietf-teas-actn-poi-applicability.txt (hunk 3 above)
Comment (italobusi) [reply draft-ietf-teas-actn-poi-applicability.txt:2556]: **2023-07-04 TEAS Call**

Agreed to clarify that the gap is about a missing attribute and not the whole configuration of LAG

**Comment by italobusi** [comment]
**[2023-07-04 ACTN POI Call](https://github.com/FabioPeruzzini/actn-poi/blob/master/minutes/minutes-2023-07-04.md)**

Most of the PR review comments are editorial and agreed

- [ ] @italobusi update the draft based on the PR review comments adding also the security consideration text provided by Daniel in issue #41

File: draft-ietf-teas-actn-poi-applicability.txt [hunk 5]
```diff
@@ -1332,25 +1324,46 @@ Internet-Draft                 ACTN POI                    January 2023
...
-   fixed grid or flexible-grid.
+   The WSON Topology Model and the Flexi-grid Topology model can be used
+   to report the DWDM network topology (e.g., ROADM nodes and links),
+   depending on whether the DWDM optical network is based on fixed-grid
+   or flexible-grid or a mix of fixed-grid and flexible-grid.
 
-   The OTN Topology Model is used to report the OTN network topology
-   (e.g., OTN switching nodes and links), when the OTN switching layer
-   is deployed within the optical domain.
+   It is worth noting that, as described in xxx, a fixed-grid can also
```
Comment (italobusi) [inline draft-ietf-teas-actn-poi-applicability.txt:1332]: Appendix I of [ITU-T_G.694.1]

File: draft-ietf-teas-actn-poi-applicability.txt [hunk 6]
```diff
@@ -3203,6 +3221,89 @@ Internet-Draft                 ACTN POI                   November 2022
...
+   muxponder's trunk (or DWDM) port and client ports. For example, there
+   might be a 100Gb/s trunk port shared by ten 10GE client ports.
+
+   The controller, when asked to set a 10GE client connectivity service
+   between two muxponder's client ports, needs first to check whether
+   there is already an existing DWDM tunnel between the two muxponders
+   and then take different actions:
+
+   1. if the DWDM tunnel already exists, the controller needs only to
+      enable the 10GE client ports to espablish the 10GE client
```
[the remark this answers is not in the document]
Comment (italobusi) [reply draft-ietf-teas-actn-poi-applicability.txt]: Fixed in https://github.com/IETF-TEAS-WG/actn-poi/pull/114/commits/65ec90d608f752d341a829c66e428d418c45da90

File: draft-ietf-teas-actn-poi-applicability.txt [hunk 7]
```diff
@@ -3203,6 +3221,89 @@ Internet-Draft                 ACTN POI                   November 2022
...
+      connectivity service;
+
+   2. if the DWDM tunnel does not exist, the controller has to first
+      establish the DWDM tunnel, finding a proper optical path matching
+      the optical parameters of the two muxponders' trunk ports (e.g.,
+      an OTSi carrying an OTU4), and then enable the 10GE client ports
+      to establish the 10GE client connectivity service.
+
+   Since multiple client connectivity services are sharing the same DWDM
+   tunnel, a multiplexing label shall assigned to each client
```
[the remark this answers is not in the document]
Comment (italobusi) [reply draft-ietf-teas-actn-poi-applicability.txt]: Fixed in https://github.com/IETF-TEAS-WG/actn-poi/pull/114/commits/65ec90d608f752d341a829c66e428d418c45da90

File: draft-ietf-teas-actn-poi-applicability.txt [hunk 8]
```diff
@@ -3203,6 +3221,89 @@ Internet-Draft                 ACTN POI                   November 2022
...
+   connectivity service. The multiplexing label can either be a standard
+   label (e.g., an OTN timeslot) or a vendor-specific label. The
+   multiplexing label can be either configurable (flexible
+   configuration) or assigned by design to each muxponder's client port
+   (fixed configuration). In the former case, any muxponder client port
+   can be connected with any other client port of the peer muxponder
+   (for example client port 1 on one muxponder can be connected with
+   cient port 5 on the peer muxponder) while in the latter case only
+   clinet ports with the same port number can be connected (for example
```
[the remark this answers is not in the document]
Comment (italobusi) [reply draft-ietf-teas-actn-poi-applicability.txt]: Fixed in https://github.com/IETF-TEAS-WG/actn-poi/pull/114/commits/65ec90d608f752d341a829c66e428d418c45da90

File: draft-ietf-teas-actn-poi-applicability.txt [hunk 9]
```diff
@@ -1493,25 +1450,35 @@ Internet-Draft                 ACTN POI                   November 2022
...
-Peruzzini et al.         Expires May 22, 2023                 [Page 28]
-

-Internet-Draft                 ACTN POI                   November 2022
+   The TE Topology Model, TE Packet Topology Model and MPLS-TE Topology
+   Model are used together to report the MPLS-TE network topology, as
+   described in [MPLS-TE-TOPO].
 
+   As described in [L3-TE-TOPO], the relationship  between the IP
+   network topology and the MPLS-TE network topology dependes on whether
```
[the remark this answers is not in the document]
Comment (italobusi) [reply draft-ietf-teas-actn-poi-applicability.txt]: Fixed in https://github.com/IETF-TEAS-WG/actn-poi/pull/114/commits/65ec90d608f752d341a829c66e428d418c45da90

File: draft-ietf-teas-actn-poi-applicability.txt (hunk 1 above)
Comment (italobusi) [reply draft-ietf-teas-actn-poi-applicability.txt]: Fixed in https://github.com/IETF-TEAS-WG/actn-poi/pull/114/commits/65ec90d608f752d341a829c66e428d418c45da90

File: draft-ietf-teas-actn-poi-applicability.txt (hunk 2 above)
Comment (italobusi) [reply draft-ietf-teas-actn-poi-applicability.txt:1361]: Fixed in https://github.com/IETF-TEAS-WG/actn-poi/pull/114/commits/65ec90d608f752d341a829c66e428d418c45da90

File: draft-ietf-teas-actn-poi-applicability.txt (hunk 3 above)
Comment (italobusi) [reply draft-ietf-teas-actn-poi-applicability.txt]: Fixed in https://github.com/IETF-TEAS-WG/actn-poi/pull/114/commits/65ec90d608f752d341a829c66e428d418c45da90

File: draft-ietf-teas-actn-poi-applicability.txt (hunk 4 above)
Comment (italobusi) [reply draft-ietf-teas-actn-poi-applicability.txt]: Fixed in https://github.com/IETF-TEAS-WG/actn-poi/pull/114/commits/65ec90d608f752d341a829c66e428d418c45da90

File: draft-ietf-teas-actn-poi-applicability.txt (hunk 5 above)
Comment (italobusi) [reply draft-ietf-teas-actn-poi-applicability.txt]: Fixed in https://github.com/IETF-TEAS-WG/actn-poi/pull/114/commits/65ec90d608f752d341a829c66e428d418c45da90

**Pull request merged**